src/Core/Grid/Definition/GridDefinition.php

```diff
@@ -68,28 +68,36 @@ final class GridDefinition implements GridDefinitionInterface
      */
     private $filters;
 
+    /**
+     * @var string
+     */
+    private $filtersUuid;
```

Comment:
Why is it called `Uuid`? At first, I though it's [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier), but it's not. :/

Reply:
I find `uuid` "prettier" when dealing with string identifiers, I agree that it doesn't respect the UUID norm.. Maybe it's problematic indeed, the idea is to have a unique identifier per Filters.

This should change, I'm still searching for a better implementation.. I need a `gridId` or a `filtersUuid` to match the filters to their respective grid, and to define the "scope" in the requests to avoid collisions with multiple grids parameters.